app/code/Magento/Contact/etc/webapi.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
+    <route url="/V1/contact" method="POST">
+        <service class="Magento\Contact\Api\ContactInterface" method="send" />
```

Comment:
It will be great if PR will be contains API-functional tests.
This kind of tests are absolutely easy (like unit) and doesn't require a large entry threshold

A couple of examples,
https://github.com/magento/inventory/blob/2.3-develop/app/code/Magento/InventoryApi/Test/Api/StockRepository/CreateTest.php#L45

https://github.com/magento/magento2/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddSimpleProductToCartTest.php